benchmarks/TFLite/linux-riscv.cmake

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+################################################################################
+#                                                                              #
+# Default benchmark configurations                                             #
+#                                                                              #
+# Each suite benchmarks a list of modules with configurations specifying a     #
+# target architecture and runtime characteristics (e.g. threads/cores). These  #
+# benchmarks only configure IREE translation and runtime flags for the target  #
+# architecture and do *not* include any non-default flags. No non-default      #
+# flags should be added here.                                                  #
+#                                                                              #
+################################################################################
+
+set(LINUX_RV64_GENERIC_CPU_TRANSLATION_FLAGS
+  "--iree-input-type=tosa"
+  "--iree-llvm-target-triple=riscv64"
+  "--iree-llvm-target-cpu=generic-rv64"
+  "--iree-llvm-target-abi=lp64d"
+  "--iree-llvm-target-cpu-features=+m,+a,+f,+d,+v"
+  "--riscv-v-vector-bits-min=256"
+  "--riscv-v-fixed-length-vector-lmul-max=8"
+)
+
+set(LINUX_RV32_GENERIC_CPU_TRANSLATION_FLAGS
+  "--iree-input-type=tosa"
+  "--iree-llvm-target-triple=riscv32-pc-linux-elf"
+  "--iree-llvm-target-cpu=generic-rv32"
+  "--iree-llvm-target-abi=ilp32"
+  "--iree-llvm-target-cpu-features=+m,+a,+f,+zvl512b,+zve32x"
+  "--riscv-v-vector-bits-min=512"
+  "--riscv-v-fixed-length-vector-lmul-max=8"
+)
+
+# CPU, Dylib-Sync, RV64-Generic, full-inference
+iree_benchmark_suite(
+  MODULES
+    "${DEEPLABV3_FP32_MODULE}"
+    "${MOBILESSD_FP32_MODULE}"
+    "${POSENET_FP32_MODULE}"
+    "${MOBILEBERT_FP32_MODULE}"
+    "${MOBILENET_V2_MODULE}"
+    "${MOBILENET_V3SMALL_MODULE}"
```

Comment:
We should only be tracking MobileNetV1 (rv64, fp32), MobileBert (rv64, fp32), DeepLabV3 (rv64, fp32) and Person Detection (rv32, quantized) for now (see [#9167](https://github.com/google/iree/issues/9167)) but we don't have to add them all as part of this PR. I would add just one for now. We can add the rest incrementally and consider more for rv32, but first we have to figure out if they are still compiling successfully for RISC-V (we are not tracking any of this) and know more about the limitations to run them on an FPGA or any other mean. I don't think we can have a large number of models for the time being due to execution constraints.